### PR TITLE
chore: sync governance files

### PR DESCRIPTION
**Pull request by HofmeisterAn**
Automated sync for OSS governance files

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
 # Contributing
 
-You are thinking about contributing to Testcontainers? Awesome, it’s absolutely appreciated. When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change. Or pick an already existing issue.
+Please see the [main contributing guidelines](./docs/contributing.md).
 
-## Guidelines
+There are additional docs describing [contributing documentation changes](./docs/contributing_docs.md).
```
Comment (mdelapenya) [inline CONTRIBUTING.md:5]: I think this contributing docs is missing

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        # Specify which users and teams can dismiss pull request reviews. Pass an empty dismissal_restrictions object to disable. User and team dismissal_restrictions are only available for organization-owned repositories. Omit this parameter for personal repositories.
+        dismissal_restrictions:
+          users: []
+          teams: [dotnet-team]
+      # Required. Require status checks to pass before merging. Set to null to disable
+      required_status_checks:
+        # Required. Require branches to be up to date before merging.
+        strict: true
+        # Required. The list of status checks to require in order to merge into this branch
+        contexts: ["analyze (csharp)", "build (ubuntu-22.04)", "build (windows-2022)"]
```
Comment (mdelapenya) [inline .github/settings.yml:177]: I love the GH checks part!

File: CONTRIBUTING.md (hunk 1 above)
Comment (HofmeisterAn) [reply CONTRIBUTING.md:5]: Thanks. I will add it. I created the PR by mistake aynway 😬

**Comment by HofmeisterAn** [comment]
I will create a new PR that contains `contributing_docs.md` and the Node.js logo.

**Pull request closed without merging**